Sources/SwifterSwift/UIKit/UINavigationControllerExtensions.swift

```diff
@@ -31,6 +31,17 @@ public extension UINavigationController {
         pushViewController(viewController, animated: true)
         CATransaction.commit()
     }
+    
+    /// SwifterSwift: Push ViewController with hide Bottom Bar.
+    ///
+    /// - Parameters:
+    ///   - viewController: viewController to push.
+    ///   - hideBottomBar: set this true parameter to hide bottom bar (like a tab bar).
+    
+    func pushViewController(_ viewController: UIViewController, hideBottomBar:Bool, animated:Bool) {
```

Comment:
Please be aware of spaces and change from `hideBottomBar` to `hidesBottomBar`
```suggestion
    /// SwifterSwift: Pushes a view controller while hiding or showing the bottom bar.
    ///
    /// - Parameters:
    ///   - viewController: The view controller to push.
    ///   - hidesBottomBar: If `true`, hides the bottom bar (e.g. tab bar).
    ///   - animated: Specify `true` to animate the transition.
    func pushViewController(_ viewController: UIViewController, hidesBottomBar: Bool, animated: Bool) {
```